data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -5,6 +5,406 @@
   "description": "Configuration schema for the crypto library subsection.",
   "type": "object",
   "properties": {
-    "additionalProperties": true
-  }
+    "cryptoConnectionFactory": {
+      "description": "Settings for database connections factory's cache of EntityManagerFactory(s)",
+      "type": "object",
+      "default": {},
+      "properties": {
```

Comment:
Could nest this under `cache` to align with `signingService`

Reply:
The service which provides the connections is more global than the signing service only, it's used in other places